joomla/joomla-cms
# [4.0] Improve list view menus

Pull Request for Issue #22093 - Another possible solution
It replaces my  former PR https://github.com/joomla/joomla-cms/pull/20787 which can be closed.

### Summary of Changes
The link to the respective menuitems is moved to an own column.

### Testing Instructions
Compare menu overview before and after patch. Make sure that links go to the correct menu items and that ACL permissions are respected

### Actual vs.  Expected result
![menu-before-after](https://user-images.githubusercontent.com/1035262/45303318-dce90700-b515-11e8-965d-427249ec16f0.PNG)

### Documentation Changes Required
yes
